index.html

```diff
@@ -418,39 +415,21 @@ <h2>The <dfn>IdleDeadline</dfn> interface</h2>
     <h2>Processing</h2>
     <section>
       <h2>Start an idle period algorithm</h2>
-      <p>The <dfn>start an idle period algorithm</dfn>, which is called
-      by the <a data-cite="html#event-loop-processing-model">event loop processing model</a> when it determines that
-      the <a>event loop</a> is otherwise idle:</p>
+      <p>To <dfn class="export">start an idle period</dfn> given {{Window}} <var>window</var> and
+      an algorithm that returns a {{DOMHighResTimeStamp}} <var>getDeadline</var>: [[HR-TIME]]
```

Comment:
Nit: Matter of editorial preference, but I tend not to add dangling references like ": [[HR-TIME]]" around the place... but it's up to you.